app/queries/concerns/lazy_comparable_query.rb

```diff
@@ -0,0 +1,42 @@
+module AhoyCaptain
+  module LazyComparableQuery
+    class LazyComparison
```

Comment:
I figured that because it's 1 extra query (not 6, like DashboardPresenter), fighting with execute_grouped_calculation is not a hill I want to die on.

this gives us the convenience of ComparableQuery.